beacon/utils/db.py

```diff
@@ -446,3 +446,145 @@ def fetch_biosamples_by_biosample(qparams_db, datasets, authenticated):
 
 def fetch_biosamples_by_individual(qparams_db, datasets, authenticated):
     return _fetch_biosamples(qparams_db, datasets, authenticated, individual_stable_id=qparams_db.targetIdReq)
+
+
+def count_variants_by_variant(qparams_db, datasets, authenticated):
+    return _count_variants(qparams_db, datasets, authenticated, variant_id=qparams_db.targetIdReq)
+
+def count_variants_by_biosample(qparams_db, datasets, authenticated):
+    return _count_variants(qparams_db, datasets, authenticated, biosample_stable_id=qparams_db.targetIdReq)
+
+def count_variants_by_individual(qparams_db, datasets, authenticated):
+    return _count_variants(qparams_db, datasets, authenticated, individual_stable_id=qparams_db.targetIdReq)
+
+def count_individuals_by_variant(qparams_db, datasets, authenticated):
+    return _count_individuals(qparams_db, datasets, authenticated, variant_id=qparams_db.targetIdReq)
+
+def count_individuals_by_biosample(qparams_db, datasets, authenticated):
+    return _count_individuals(qparams_db, datasets, authenticated, biosample_stable_id=qparams_db.targetIdReq)
+
+def count_individuals_by_individual(qparams_db, datasets, authenticated):
+    return _count_individuals(qparams_db, datasets, authenticated, individual_stable_id=qparams_db.targetIdReq)
+
+def count_biosamples_by_variant(qparams_db, datasets, authenticated):
+    return _count_biosamples(qparams_db, datasets, authenticated, variant_id=qparams_db.targetIdReq)
+
+def count_biosamples_by_biosample(qparams_db, datasets, authenticated):
+    return _count_biosamples(qparams_db, datasets, authenticated, biosample_stable_id=qparams_db.targetIdReq)
+
+def count_biosamples_by_individual(qparams_db, datasets, authenticated):
+    return _count_biosamples(qparams_db, datasets, authenticated, individual_stable_id=qparams_db.targetIdReq)
+
+@pool.asyncgen_execute
```

Comment:
As these `count_xxx` functions only return a single value, there is no need to make them an async generator, they just need to be async coroutines. 
On the contrary, the `fetch_xxx` functions can return many rows, and we want to spit out the values little at a time, so we make them async generator.